Preparation for AA week 2

shiquan-lim opened the pull request:
Made changes to text response viewing. Now drops down similar to a menu when clicked. This is to adjust to limitation to md-tooltip as used previously.

Also contain minor bugfixes.

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into #125-change-linkpattern-to-regex

shiquan-lim committed:
Only show textresponse submission in tooltip if the answer was truncated to begin with

shiquan-lim committed:
Merge branch 'master' of https://github.com/singpath/classmentors

merge upstream

shiquan-lim committed:
Fixing profile edit challenge. Currently auth data is not updated, causing userProfile data to be over written when a cleanObj is called from Auth

Commit:
#125 changed link pattern to regex

#125 changed link pattern from static string to regex

sheryltohxn committed:
Close #146  Changed incorrect nav bar title

sheryltohxn committed:
Close #136 changed spfAlert messages

Changed from 'task' to 'challenge'
Changed message for survey completion

shiquan-lim committed:
Fixed issues with challenge table view. Viewing long text responses now shows as a menu-like view. Also ammended text area input view

sheryltohxn committed:
close #161  resolved conflict button

sheryltohxn committed:
close #166 new event, mcq progress bug

shiquan-lim committed:
Merge branch 'master' of https://github.com/shiquan-lim/classmentors

Merge master

shiquan-lim committed:
Merge pull request #165 from shiquan-lim/#161-resolve-conflict-button

close #161  resolved conflict button

shiquan-lim committed:
Merge pull request #167 from shiquan-lim/#166-new-event-mcq-progress-bug

close #166 new event, mcq progress bug

shiquan-lim committed:
Resolve conflicts

shiquan-lim committed:
Merge pull request #158 from shiquan-lim/#125-change-linkpattern-to-regex

125 change linkpattern to regex

shiquan-lim committed:
Merge branch 'master' of https://github.com/singpath/classmentors

Merge upstream

SingaporeClouds commented:
It is no longer possible for event owners to create new text challenges with this PR.

shiquan-lim committed:
Fixed challenge creation bug

shiquan-lim commented:
Fixed. We had added some code to allow link pattern to be based off regular expression, instead of static string matching. This had broken the code for other challenge creation. It should not have affected other parts of the application.
